sections/literature_review/main.tex

```diff
@@ -0,0 +1,91 @@
+allowing the players (firms) to choose capacities. 
+A main result from this paper was that when both firms operate independently as
+a monopoly, the equilibria are socially optimal, but this is not the case when
+the firms operate together.
+Another extension of~\cite{FirmCompetition} was introduced 
+in~\cite{FirmCompetitionExtension} where a long-run version of the competition 
+was considered that also had capacity as a decision variable.
+In~\cite{knight2017measuring} a normal form game is built that is informed by a 
+two-dimensional Markov chain in order to model interactions between critical
+care units.
+An additional paper that focuses on competition is~\cite{fan2009short} where
+the authors created a competition between two sellers where seller 1 supplies 
+a product instantly and seller 2 is modelled as a make-to-order M/M/1 queue.
+The game that is played requires the two sellers to make a choice on the price 
+of the product and then seller 2 to set a capacity that guarantees a maximum 
+expected delay.
+In this paper, while giving some consideration to equilibrium behaviour,
+similar to the work of~\cite{FirmCompetition, FirmCompetition2}, emergent
+behaviour is more precisely addressed by considering learning algorithms like
+asymmetric replicator dynamics~\cite{fudenberg1998theory}.
+
+In the above models, the players are attempting to increase their share of 
+individuals choosing to queue.
+In public healthcare type settings~\cite{knight2013selfish}, this is not 
+necessarily the case. 
+Players might not necessarily aim to maximise the so called `market share'.
+Rather, the overall service needs to be considered as players aim to minimise
+their experienced congestion.
+In~\cite{sadat2015can} a healthcare application was studied where patients 
+could choose between two hospitals, where a utility function is derived that is
+based on patients' perceived quality of life.
+How individual behaviour based on congestion impacts the efficiency of public 
+services is investigated in~\cite{knight2013selfish}.
+The authors place the individuals' choices between different public services 
+within the formulation of routing games and measure inefficiencies using a 
+concept known as the price of anarchy (PoA)~\cite{koutsoupias1999worst}.
+They show that the price of anarchy increases with worth of service and that is
+low for systems with insufficient capacities.
+In~\cite{deo2011centralized} the authors study the network effect of ambulance 
+diversion by proposing a non-cooperative game between two EDs that are modelled
+as a queueing network.
+Each ED's objective is to minimise its own waiting time and chooses a diversion
+threshold based on the patients it has.
+In equilibrium both EDs choose to divert ambulances in order to avoid getting
+arrivals from the other ED.
+In this paper this concept is extended by allowing the ambulance service to 
+decide how to distribute its patients among the two EDs.
```

Comment:
I would more explicitly say how this paper is different:

**Players are both hospitals and customers of the hospitals. As opposed to the previous models which are one or the other. So the novelty in a way is combining both aspects.**